state-manager-frontend/src/components/StatesByRunId.tsx

```
@@ -0,0 +1,331 @@
+  }
+
+  return (
+    <div className="w-full max-w-6xl mx-auto p-6">
+      <div className="flex items-center justify-between mb-6">
+        <h2 className="text-2xl font-bold text-gray-800">States by Run ID</h2>
+        <div className="flex items-center space-x-2">
+          {selectedRunId && (
+            <button
+              onClick={() => setShowGraph(!showGraph)}
+              className="flex items-center space-x-2 px-4 py-2 bg-green-600 text-white rounded-lg hover:bg-green-700 transition-colors"
+            >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)